src/backend/expungeservice/models/helpers/charge_classifier.py

```diff
@@ -163,3 +165,8 @@ def _person_felony(statute):
             # In this case the type eligibility needs more analysis. The condition is checked again in the charge object's type eligibility method.
         else:
             return False
+
+    @staticmethod
+    def _sex_crime(statute):
+        if statute in SexCrimes.statutes or statute in SexCrimes.romeo_and_juliet_exceptions:
```

Comment:
Here's the style we're using elsewhere for this:

```suggestion
        if statute in SexCrimes.statutes + SexCrimes.romeo_and_juliet_exceptions:
```

Reply:
done